### PR TITLE
{phys}[foss/2019a] Octopus v9.1 plus dependencies (WIP)

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 4 out of 6 (4 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/66ae5de0a32174fcb8403cbc5cb5f14b for a full test report.

**Comment by vanzod** [comment]
It looks like the website hosting the `Octopus` sources is currently down.

**Comment by boegel** [comment]
Octopus website has been down for a while now, see https://gitlab.com/octopus-code/octopus/issues/218 .

Should we consider downloading the sources from GitLab instead?

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/8a0db499e8dbcc0475f72fa781543352 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/54c62e48f73a58b9feb25c9e7af93654 for a full test report.

File: easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.6.2-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/opencollab/arpack-ng/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['673c8202de996fd3127350725eb1818e534db4e79de56d5dcee8c00768db599a']
+
+builddependencies = [('Autotools', '20180311')]
+
+preconfigopts = "sh bootstrap && "
+configopts = '--enable-mpi --with-pic --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
```
Comment (boegel) [inline easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.6.2-foss-2019a.eb]: existing `arpack-ng-3.6.2-intel-2018a.eb` easyconfig also has `--enable-icb`?

File: easybuild/easyconfigs/o/Octopus/Octopus-9.1-foss-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+easyblock = 'ConfigureMake'
+
+name = 'Octopus'
+version = '9.1'
+
+homepage = 'http://octopus-code.org'
```
Comment (smoors) [inline easybuild/easyconfigs/o/Octopus/Octopus-9.1-foss-2019a.eb]: @migueldiascosta please use https here?

File: easybuild/easyconfigs/e/ETSF_IO/ETSF_IO-1.0.4-foss-2019a.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'ETSF_IO'
+version = '1.0.4'
+
+homepage = 'http://www.etsf.eu/resources/software/libraries_and_tools'
```
Comment (smoors) [inline easybuild/easyconfigs/e/ETSF_IO/ETSF_IO-1.0.4-foss-2019a.eb]: @migueldiascosta change to https? also this particular page seems to be down for the moment.

File: easybuild/easyconfigs/p/PSolver/PSolver-1.7.7-foss-2019a.eb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'PSolver'
+version = '1.7.7'
+
+homepage = 'http://bigdft.org/Wiki/index.php?title=BigDFT_website'
```
Comment (smoors) [inline easybuild/easyconfigs/p/PSolver/PSolver-1.7.7-foss-2019a.eb]: the https site gives a certificate error.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 4 out of 5 (4 easyconfigs in this PR)
node359.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e29ad99aa46158764d4ed9b692354850 for a full test report.

**Comment by smoors** [comment]
@migueldiascosta the test `spglib_test` failed.

**Comment by migueldiascosta** [comment]
@smoors hm, I can reproduce that if I also build on Skylake

I can get the `spglib` test to pass if I set `-march` to `skylake` (instead of `skylake-avx512` which is what `native` sets), but then the `octopus` own tests still fail...

**Comment by migueldiascosta** [comment]
superseded by #14229 and #15167

**Pull request closed without merging**